src/components/TokenInput.tsx

```diff
@@ -59,13 +63,13 @@ function TokenInput({
   }
 
   function onChangeInput(e: React.ChangeEvent<HTMLInputElement>): void {
-    const { decimals } = TOKENS_MAP[symbol]
+    const { decimals } = TOKENS_MAP[symbol] || 0
```

Comment:
this destructing should fail, since this would evaluate to `const {decimals} = 0`. Instead do `const {decimals = 0} = TOKENS_MAP[symbol]`

Reply:
@ug02fast , `const { decimals } = TOKENS_MAP?.[symbol]` also gives error, when there is no token on `POOL_MAP`, because `TOKENS_MAP?.[symbol]` will be `undefined`.

Reply:
Ok thanks for checking.